packages/frontend/src/components/PageLayout/PageLayout.tsx

```diff
@@ -42,6 +42,13 @@ const PageLayoutContent: React.FC<PageLayoutContentProps> = memo(
   },
 );
 
+const Background: React.FC<{ children: React.ReactNode; isCompany: boolean }> = ({ children, isCompany }) => {
+  const { inSelectionMode } = useSelectedDialogs();
+  const className = (isCompany ? 'isCompany' : '') + (inSelectionMode ? styles.inSelectionMode : '');
```

Comment:
Kanskje bruke `classnames` for dette? eksemepl
https://github.com/digdir/dialogporten-frontend/blob/main/packages/frontend/src/components/Avatar/Avatar.tsx#L22